src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,54 @@
+import { ArrowForwardIcon } from '@chakra-ui/icons'
+import { Flex } from '@chakra-ui/react'
+import { useCallback } from 'react'
+import { useHistory } from 'react-router'
+import { Main } from 'components/Layout/Main'
+import { DashboardHeaderWrapper } from 'pages/Dashboard/components/DashboardHeader/DashboardHeaderWrapper'
+import { WalletBalance } from 'pages/Dashboard/components/DashboardHeader/WalletBalance'
+import { ProfileAvatar } from 'pages/Dashboard/components/ProfileAvatar/ProfileAvatar'
+
+import { PriceTable } from './PriceTable'
+
+const paddingTop = { base: 'calc(env(safe-area-inset-top) + 2rem)', md: '4.5rem' }
+const marginTop = { base: 0, md: '-4.5rem' }
+const activeStyle = { opacity: 0.5 }
+const pageProps = { paddingTop: 0 }
+
+export const Home = () => {
+  const history = useHistory()
+  const handleWalletClick = useCallback(() => {
```

Comment:
Not sure if that's the intent but this will rug the avatar edit


https://github.com/shapeshift/web/assets/17035424/f4fe2a81-add0-4958-ad26-dba18bf8d863

Reply:
This is the intention, this is just a button to link to the wallet page. I'll explicitly disable the avatar button.